src/view3d.cpp

```diff
@@ -355,18 +518,18 @@ void View3D::drawMeshes(const TransformState& transState,
         meshFaceShader.setUniformValue("lightDir_eye", lightDir.x, lightDir.y, lightDir.z);
         for (size_t i = 0; i < geoms.size(); ++i)
             geoms[i]->drawFaces(meshFaceShader, transState);
-        meshFaceShader.release();
+        // meshFaceShader.release();
```

Comment:
I think remove it everywhere if that's the right thing to do.

My understanding is that `glUseProgram(0)` is harmless enough, and might be worthwhile in the sense that it prevents state from one part of the program from unexpectedly flowing on to another via the global OpenGL context: It's a way to mitigate the fact that the context is a global variable, to some extent.

Other than that, it's arguably an OpenGL API call that we don't need, so make things strictly less efficient.

Is there something else going on here that I don't understand?